internal/runners/activate/activate.go

```diff
@@ -81,7 +62,12 @@ func (r *Activate) run(params *ActivateParams, activatorLoop activationLoopFunc)
 		}
 	}
 
-	go sendProjectIDToAnalytics(params.Namespace, filepath.Join(targetPath, constants.ConfigFileName))
+	// Send google analytics event with label set to project namespace
+	names, fail := project.ParseNamespaceOrConfigfile(params.Namespace.String(), filepath.Join(targetPath, constants.ConfigFileName))
+	if fail != nil {
+		logging.Debug("error resolving namespace: %v", fail.ToError())
+	}
+	analytics.EventWithLabel(analytics.CatRunCmd, "activate", names.String())
```

Comment:
`params.Namespace.String()` is already what you want, no need to parse the config file again.

Reply:
I think that `params.Namespace.String()` can be empty as the namespace argument to the activate command is optional:
https://github.com/ActiveState/cli/blob/master/cmd/state/internal/cmdtree/activate.go#L46

So, if I just run `state activate` without any arguments, the namespace needs to be sourced from the as.yaml file.